Library/Homebrew/formula_installer.rb

```diff
@@ -235,6 +235,10 @@ def install
       end
       raise CannotInstallFormulaError, message
     end
+    if formula.opt_prefix.directory? &&
+       formula.pkg_version < (v = Keg.new(formula.opt_prefix.resolved_path).version)
+      opoo "Downgrading #{formula.name} from #{v} to #{formula.pkg_version}."
```

Comment:
Sorry, rather than downgrading here would be good to check the tab in the bottle and see if the formula file exists and that one has a newer version.

Reply:
Ah, got it. Sorry I misunderstood. I've updated the warning.